charts/dashboards/realtime.json

```diff
@@ -15,8 +15,7 @@
   "editable": true,
```

Comment:
For all dashboards: put utilization above request so that the colors are consistent